scene/main/viewport.cpp

```diff
@@ -3955,7 +3955,7 @@ void SubViewport::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_clear_mode"), &SubViewport::get_clear_mode);
 
 	ADD_PROPERTY(PropertyInfo(Variant::VECTOR2, "size"), "set_size", "get_size");
```

Comment:
This is still missing. `size` is a `Size2i` like `size_2d_override`, so its type should be reflected here.